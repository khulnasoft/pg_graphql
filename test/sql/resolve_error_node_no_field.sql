begin;

    create table account(
        id int primary key,
        parent_id int references account(id)
    );


    select graphql.resolve($$
    {
      accountCollection {
        edges {
          cursor
          node {
            parent {
              dneField
            }
          }
        }
      }
    }
    $$);

rollback;
