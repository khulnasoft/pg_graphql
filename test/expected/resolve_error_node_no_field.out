begin;
    create table account(
        id int primary key,
        parent_id int references account(id)
    );
    select graphql.resolve($$
    {
      accountCollection {
        edges {
          cursor
          node {
            parent {
              dneField
            }
          }
        }
      }
    }
    $$);
                                        resolve                                        
---------------------------------------------------------------------------------------
 {"data": null, "errors": [{"message": "Unknown field 'dneField' on type 'Account'"}]}
(1 row)

rollback;
